print/print-apps/oereb/mainpage.jrxml

```diff
@@ -137,6 +139,7 @@
 			<textField>
 				<reportElement x="0" y="697" width="100" height="12" uuid="e0c1f775-61b3-41c4-a7ec-b8e68adeeb11">
 					<property name="com.jaspersoft.studio.unit.y" value="mm"/>
+					<printWhenExpression><![CDATA[true]]></printWhenExpression>
```

Comment:
This should also be `<![CDATA[$P{Display_Certification}]]>`, otherwise there is still the title there...